cli/turbo.json

```diff
@@ -3,19 +3,72 @@
   "extends": ["//"],
   "pipeline": {
     "build": {
+      "inputs": ["internal/**"],
```

Comment:
Not necessarily related to these changes, but in the same ballpark. Should we change our build script? Right now we're just running `make` which produces a ton of junk that's not the turbo executable. If we're being honest we should also list out all of the Rust crates that end up getting used when building our Rust and Go binary

Reply:
added all the crates and stuff (need to verify that it actually invalidates cache on the crates changing)